### PR TITLE
WASM: uint64 -> double edge case

### PR DESCRIPTION
**Pull request by Cellule**
Fix uint64 conversion to double in edge cases on x64

---
This change is [Reviewable](https://reviewable.io/reviews/microsoft/chakracore/3426)

### CODE REVIEW

**Comment by Cellule** [comment]
@dotnet-bot test this please

File: lib/Runtime/Language/Chakra.Runtime.Language.vcxproj [hunk 1]
```diff
@@ -256,6 +256,9 @@
     <ClInclude Include="WebAssemblySource.h" />
   </ItemGroup>
   <ItemGroup>
+    <MASM Include="amd64\JavascriptConversionA.asm">
```
Comment (Penguinwizzard) [inline lib/Runtime/Language/Chakra.Runtime.Language.vcxproj]: Does this need MSBuildThisFileDirectory?

Comment (Penguinwizzard) [inline lib/Runtime/Language/Chakra.Runtime.Language.vcxproj]: Does this need MSBuildThisFileDirectory?

File: lib/Runtime/Language/Chakra.Runtime.Language.vcxproj.filters [hunk 2]
```diff
@@ -157,6 +157,7 @@
     <MASM Include="$(MSBuildThisFileDirectory)amd64\JavascriptOperatorsA.asm">
       <Filter>amd64</Filter>
     </MASM>
+    <MASM Include="amd64\JavascriptConversionA.asm" />
```
Comment (Penguinwizzard) [inline lib/Runtime/Language/Chakra.Runtime.Language.vcxproj.filters]: Does this need MSBuildThisFileDirectory?
